scripts/bfasst/compare_waveforms/Templates/structs.py

```diff
@@ -0,0 +1,35 @@
+"""A data structure used to store the list of inputs, outputs, and their respective bits."""
```

Comment:
Note that you don't need to pre-declare data structures, you should be able to access them as necessary. see `setdefault()`